src/snmalloc/backend_helpers/pagemap.h

```diff
@@ -325,4 +325,105 @@ namespace snmalloc
       body[p >> SHIFT] = t;
     }
   };
+
+  /**
+   * This is the backend's interface to the page map. It takes a concrete page
```

Comment:
```suggestion
* This is a generic implementation of the backend's interface to the page map. It takes a concrete page
```
This isn't the interface, this is one thing that you can choose to use to implement the interface.  You are also free to use something else.